src/Helpers/SpellHelper.php

```diff
@@ -114,7 +121,37 @@ public function getSelfSpells(): Collection
 //                'duration' => 10,
 //                'cooldown' => 22, // todo
 //            ],
-            // todo: racial
+            $racialSpell
+        ]);
+    }
+
+    public function getRacialSelfSpells(): Collection
+    {
+        return collect([
+            [
+                'name' => 'Crusade',
+                'description' => '+5% offensive power, allows you to kill Undead',
+                'key' => 'crusade',
+                'mana_cost' => 5,
+                'duration' => 12,
+                'races' => collect(['Human', 'Nomad']),
```

Comment:
The races values here can just be a regular PHP array. I use Collections mainly so I can easily filter and search through them.

Reply:
I'm quite the noob at what's in the framework and not. Sounds like a plan. Cheers!